chainer/training/triggers/interval.py

```diff
@@ -40,9 +42,16 @@ def __call__(self, trainer):
         """
         updater = trainer.updater
         if self.unit == 'epoch':
-            prev = self.count
-            self.count = updater.epoch_detail // self.period
-            return prev != self.count
+            epoch_detail = updater.epoch_detail
+            previous_epoch_detail = updater.previous_epoch_detail
```

Comment:
I am sorry to make you wait. We discussed the matter and will share you the summary within few days.